op-batcher/batcher/driver.go

```diff
@@ -597,6 +590,12 @@ func (l *BatchSubmitter) sendTransaction(ctx context.Context, txdata txData, que
 			l.queueTx(txdata, false, candidate, queue, receiptsCh)
 			return nil
 		})
+		if !goroutineSpawned {
+			// We couldn't start the goroutine because the errgroup.Group limit
+			// is already reached. Since we can't send the txdata, we have to
+			// return it for later processing.
+			l.recordFailedTx(txdata.ID(), fmt.Errorf("could not start da writing goroutine: %w", err))
```

Comment:
My only issue with this is that if we have a poll frequency of 1sec say, and we have max number of goroutines running but still 1 more frame to send, then we'll constantly be running into this `recordFailedTx` which logs an error statement and would pollute the log output. Maybe this is fine though?